Doc/library/datetime.rst

```diff
@@ -752,6 +752,13 @@ Other constructors, all class methods:
    :class:`.datetime` object.  An aware current UTC datetime can be obtained by
    calling ``datetime.now(timezone.utc)``.  See also :meth:`now`.
 
+   .. warning::
+
+      Because naive datetimes are treated by many functions as local times,
+      it is recommended that you create a timezone-aware datetime
```

Comment:
Phrasing improvement, it's preferable to avoid the usage of pronouns such as "you" when possible within technical writing, and I think "timezone specific" makes a bit more sense than "timezone-aware". 
```suggestion
      it is recommended to create a timezone specific ``datetime`` object
```

Reply:
Thanks for the review. I consulted @pganssle at the sprints and he says we should keep to aware here. I hope I have addressed the other suggestions.

Reply:
@nanjekyejoannah:
> Thanks for the review. I consulted @pganssle at the sprints and he says we should keep to aware here. I hope I have addressed the other suggestions.

Sounds good to me. If Paul prefers using "timezone-aware" over "timezone specific", I have no objections.